multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -138,15 +141,15 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	epResExportName := getResourceExportName(r.localClusterID, req, "endpoints")
 
 	cleanup := func() error {
+		err = r.handleServiceDeleteEvent(ctx, req, commonArea)
```

Comment:
Could you add some comments to explain why we do not check svcInstalled?

Reply:
Comment added
I leave it to caller to check the svcInstalled. In some rare cases, it might be better to call clean up anyway.